content/en/golden_signals.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Golden Signals
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+Golden signals contain a set of criteria that provide a wide view of a service from a user’s perspective . It is the gold standard for monitoring a web application’s performance.
+
+## Problem it addresses
+Old monitoring tools are much suitable as these were created to monitor machines, not microservices. On the other hand, most of the modern monitoring systems offer a huge variety of parameters for various purposes, making it easy to drown in and lose focus of what really is relevant for the application.
```

Comment:
I think we should focus on the problem that occurs if we don't have that type of data. Without having latency, traffic, errors, and saturation data, operators are flying blind. What does typically goes wrong when you don't have that data?